glue_jupyter/bqplot/common/tools.py

```diff
@@ -398,6 +412,17 @@ def update_selection(self, *args):
                 yc = y.mean()
                 rx = abs(x[1] - x[0])/2
                 ry = abs(y[1] - y[0])/2
+                # Avoid drawing invisible shape.
+                if isinstance(self._roi, CircularROI):
+                    if rx < 1:
+                        rx = self._roi.radius
+                    if ry < 1:
+                        ry = self._roi.radius
+                elif isinstance(self._roi, EllipticalROI):
+                    if rx < 1:
+                        rx = self._roi.radius_x
+                    if ry < 1:
+                        ry = self._roi.radius_y
```

Comment:
```suggestion
                # Avoid drawing invisible shape.
                rx = max(abs(x[1] - x[0])/2, 1.0)
                ry = max(abs(y[1] - y[0])/2, 1.0)
```
Same as for rectangle